Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -479,6 +479,12 @@ class NewTabPageViewController: UIViewController, Themeable {
                 inNewTab: false,
                 switchingToPrivateMode: false
             )
+        case .bravePartnerLearnMoreTapped:
+            delegate?.navigateToInput(
+                BraveUX.braveTodayPartnersURL.absoluteString,
+                inNewTab: false,
```

Comment:
I wonder if it would not be better to show in new tab

Reply:
Good point… Im not sure to be honest since we show "More Brave Offers" in the current tab when tapped.
 
@mattmcalister @jamesmudgett any particular preference?

Reply:
Talked to James and Matt via Slack and decision was to keep them opening in the same tab